views/layouts/main.php

```diff
@@ -19,6 +20,7 @@
 use yii\bootstrap\NavBar;
 use yii\widgets\Breadcrumbs;
 use app\assets\AppAsset;
+use rmrevin\yii\fontawesome\FAS;
```

Comment:
why did you add this import ?

Reply:
In order to add the annotation flag I had to add FontAwesome icons. It is the css dependency.